packages/dependency-royalty-management/contracts/RoyaltyManager.sol

```diff
@@ -69,13 +69,13 @@ contract RoyaltyManager is AccessControlUpgradeable, IRoyaltyManager {
 
     /// @notice sets the trustedForwarder address to be used by the splitters
     /// @dev can only be called by the admin
+    /// new splitters will be deployed with this setting; existing splitters will have to apply it
```

Comment:
This is not true.

This is not a deployment setting for a splitter, the splitter just asks the RoyaltyManager what is the trusted forwarder instead of returning the local variable, there is nothing to `apply` on existing splitters?

Reply:
This comment was already there and added by @atkinsonholly but yes I'll change it.